### ISSUE TITLE
tox4: description for cli option "verbosity" could be updated

### ISSUE THREAD

**Issue by jugmac00** [tox-dev/tox#2005]
Currently you can read

> verbosity=verbose-quiet, default INFO, map 0 - CRITICAL|1 - ERROR|2 - WARNING|3 - INFO|4 - DEBUG|5 - NOTSET

see https://tox.readthedocs.io/en/rewrite/cli_interface.html#verbosity

I do not understand what `verbosity=verbose-quiet` stands for.

Also, from above description I would not know that I have to add more `v` characters to increase verbosity.

**Comment by gaborbernat** [comment]
Basically, verbosity is a play between `-q` and `-v`. Every `-v` increases the verbose counter, every `-q` increases the quiet counter. The verbosity is the difference between them. Doesn't make sense to read this on its own you need to read it together with the `-v` and `-q` documentation, hence them being in their own group.

**Comment by gaborbernat** [comment]
```
verbosity:
  verbosity=verbose-quiet, default INFO, map 0 - CRITICAL|1 - ERROR|2 - WARNING|3 - INFO|4 - DEBUG|5 - NOTSET

  -v, --verbose               increase verbosity (default: 2)
  -q, --quiet                 decrease verbosity (default: 0)
```

**Comment by jugmac00** [comment]
Thanks for your explanation. Even from the "combined" view, I wouldn't have known that you can use more than one `v` or `q` flag. But your textual explanation made it very clear... why not use something like that for the description?

### current
```
verbosity:
  verbosity=verbose-quiet, default INFO, map 0 - CRITICAL|1 - ERROR|2 - WARNING|3 - INFO|4 - DEBUG|5 - NOTSET

  -v, --verbose               increase verbosity (default: 2)
  -q, --quiet                 decrease verbosity (default: 0)
```

### suggestion 1
```
verbosity:
  every -v increases, every -q decreases verbosity level, default INFO, map 0=CRITICAL|1=ERROR|2=WARNING|3=INFO|4=DEBUG|5=NOTSET

  -v, --verbose               increase verbosity (default: 2)
  -q, --quiet                 decrease verbosity (default: 0)
```

### suggestion 2
```
verbosity:
  add -v to increase, add -q to decrease verbosity level, default INFO, map 0=CRITICAL|1=ERROR|2=WARNING|3=INFO|4=DEBUG|5=NOTSET

  -v, --verbose               increase verbosity (default: 2)
  -q, --quiet                 decrease verbosity (default: 0)
```

**Comment by gaborbernat** [comment]
I'm happy with suggestion 1 if you put in a pr.

**Comment by jugmac00** [comment]
closed by #2019
